install/kubernetes/helm/istio/charts/grafana/templates/deployment.yaml

```diff
@@ -16,24 +16,22 @@ spec:
       labels:
         app: {{ template "grafana.name" . }}
         release: {{ .Release.Name }}
+      annotations:
+        sidecar.istio.io/inject: "false"
     spec:
       containers:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.global.grafana.repository }}:{{ .Values.global.grafana.tag }}"
           imagePullPolicy: {{ .Values.imagePullPolicy }}
           ports:
             - containerPort: {{ .Values.service.internalPort }}
-          livenessProbe:
-            httpGet:
-              path: /logging
-              port: {{ .Values.service.internalPort }}
           readinessProbe:
             httpGet:
-              path: /logging
+              path: /login
```

Comment:
this looks good.  Should probably also go in the kubernetes/templates file as a separate PR if you feel up to it :)